docs/example/accordionDocs.vue

```diff
@@ -95,6 +95,20 @@
     <doc-code language="markup">
       <panel header="Dynamic Loading" src="/docs/loadContent.html#fragment" minimized></panel>
     </doc-code>
+    <br>
+
+    <h4>If <code>popup-url</code> attribute is provided, a popup button will be displayed. Upon clicking, it opens a new window to the specified url.</h4>
```

Comment:
How about:
- "will be `shown`" for consistent grammar in this docs.
- "If clicked, it opens the specified url in a new window."